src/test/java/io/stargate/sgv2/jsonapi/api/v1/util/DataApiCommandSenderBase.java

```diff
@@ -0,0 +1,73 @@
+package io.stargate.sgv2.jsonapi.api.v1.util;
+
+import static io.stargate.sgv2.jsonapi.api.v1.util.IntegrationTestUtils.getAuthToken;
+import static io.stargate.sgv2.jsonapi.api.v1.util.IntegrationTestUtils.getCassandraPassword;
+import static io.stargate.sgv2.jsonapi.api.v1.util.IntegrationTestUtils.getCassandraUsername;
+
+import io.stargate.sgv2.jsonapi.config.constants.HttpConstants;
+import io.stargate.sgv2.jsonapi.service.embedding.operation.test.CustomITEmbeddingProvider;
+import java.util.Base64;
+import java.util.Map;
+import org.eclipse.microprofile.config.ConfigProvider;
+
+/**
+ * Helper class used for constructing and sending commands to the Data API for testing purposes
+ * (usually integration tests).
+ */
+public abstract class DataApiCommandSenderBase<T extends DataApiCommandSenderBase> {
+  protected final String namespace;
+
+  protected int expectedHttpStatus = 200;
```

Comment:
Nit: use Response.Status.OK  (payback for your ninja fix on my use of 200  :) ) 

lso, maybe change the expectedHttpStatus below to use the Response.Status enum

Reply:
Fair, will change.